### PR TITLE
Tar untar doc

### PR DESCRIPTION
**Pull request by mat007**
And a few fixes

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -98,7 +103,11 @@ func glob(sources []string) ([]string, error) {
 		if err != nil {
 			return nil, err
 		}
-		paths = append(paths, matches...)
+		if matches == nil && !strings.ContainsAny(source, "*?[]^") {
```
Comment (jeanlaurent) [inline main.go]: Does this deserve a test for itself ?

File: main.go [hunk 2]
```diff
@@ -108,6 +117,9 @@ func copy(sources []string, destination string) error {
 	if err != nil {
 		return err
 	}
+	if len(sources) == 0 {
```
Comment (jeanlaurent) [inline main.go]: I guess this can be tested

File: main.go [hunk 3]
```diff
@@ -199,6 +211,9 @@ func tarFiles(dst string, srcs ...string) error {
 	if err != nil {
 		return err
 	}
+	if len(srcs) == 0 {
```
Comment (jeanlaurent) [inline main.go]: I guess this can be tested too.

File: main.go (hunk 1 above)
Comment (mat007) [reply main.go]: Yes, also thinking about it I believe we should only support `*` and `?`…

**Comment by mat007** [comment]
Added tests, PTAL!

**Pull request merged**